test/cloudant.test.js

```diff
@@ -68,6 +68,36 @@ describe('cloudant connector', function() {
           });
         };
       });
+    it('should create new instance if id provided', function(done) {
```

Comment:
Feel free to ignore since in this file other tests use the same pattern; however, in general we use imperative mood, we do not start test names with should. Please see: https://github.com/strongloop/loopback.io/blob/gh-pages/pages/en/contrib/style-guide.md#test-naming